lib/chef/knife/maas/version.rb

```diff
@@ -1,7 +1,7 @@
 class Chef
   class Knife
     module Maas
-      VERSION = "1.2.0"
+      VERSION = '1.2.0'
```

Comment:
*cough* https://github.com/chef-partners/maas *cough*

Also if you have a spare ubuntu 14.04 box sitting around these video(s) will show you how to set it up leveraging qemu/kvm. https://youtu.be/ojTTgrtl-RU